alicloud/extension_emr.go

```diff
@@ -0,0 +1,32 @@
+package alicloud
+
+const (
+	HOST_GROUP_TYPE_MASTER = string("MASTER")
+	HOST_GROUP_TYPE_CORE   = string("CORE")
+	HOST_GROUP_TYPE_TASK   = string("TASK")
+
+	CHARGE_TYPE_POSTPAID = string("PostPaid")
+	CHARGE_TYPE_PREPAID  = string("PrePaid")
+
+	PREPAID_ONE_MONTH    = string("1")
+	PREPAID_TWO_MONTHS   = string("2")
+	PREPAID_THREE_MONTHS = string("3")
+	PREPAID_FOUR_MONTHS  = string("4")
+	PREPAID_FIVE_MONTHS  = string("5")
+	PREPAID_SIX_MONTHS   = string("6")
+	PREPAID_SEVEN_MONTHS = string("7")
+	PREPAID_EIGHT_MONTHS = string("8")
+	PREPAID_NINE_MONTHS  = string("9")
+	PREPAID_ONE_YEAR     = string("12")
+	PREPAID_TWO_YEARS    = string(24)
+	PREPAID_THREE_YEARS  = string(36)
+
+	DISK_TYPE_CLOUD            = string("CLOUD")
+	DISK_TYPE_CLOUD_EFFICIENCY = string("CLOUD_EFFICIENCY")
+	DISK_TYPE_CLOUD_SSD        = string("CLOUD_SSD")
+	DISK_TYPE_LOCAL_DISK       = string("LOCAL_DISK")
+	DISK_TYPE_CLOUD_ESSD       = string("CLOUD_ESSD")
```

Comment:
Please use the common const: https://github.com/terraform-providers/terraform-provider-alicloud/blob/master/alicloud/extension_ecs.go#L97

Reply:
not the same. The api parameters are all upper cases.

Reply:
In order to keep the same usage experience, please convert them in the resource.